README.md

```diff
@@ -1,3 +1,84 @@
 # LOCK API
 
-API for LOCK.space international staking service
\ No newline at end of file
+API for LOCK.space international staking service
+
+# Documentation
+
+- [Registration](#registration)
+- [KYC](#kyc)
+- [Staking](#staking)
+- [Deposit](#deposit)
+- [Withdrawal](#withdrawal)
+- [Analytics](#analytics)
+- [Voting](#voting)
+- [Integration Example](#integration-example)
+
+# Registration
+
+1. Contact [support](support@lock.space) to registrate your wallet name
+2. Create an address on selected blockchain (e.g. DeFiChain)
+3. Get and sign message from sign-message endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_getSignMessage
+4. Register the user with the sign-up endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_signUp
+5. Now you can get your access token (with address & signature) with the sign-in endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_signIn
```

Comment:
I would hide the Swagger URLS (everywhere), e.g.
`Now you can get your access token (with address & signature) with the (sign-in endpoint)[https://dev.api.lock.space/swagger/#/Authentication/AuthController_signIn]`

Reply:
Good point. I think you meant:

`Now you can get your access token (with address & signature) with the [sign-in endpoint](https://dev.api.lock.space/swagger/#/Authentication/AuthController_signIn)`

Reply:
Ehh yes, true 😉